src/relcast.erl

```diff
@@ -897,7 +909,7 @@ get_mod_state(DB, Module, ModuleState0, WriteOpts) ->
                     {ok, KeyTreeBin} ->
                         KT = binary_to_term(KeyTreeBin),
                         do_deserialize(Module, ModuleState0, ?stored_key_prefix, KT, DB);
-                    not_found ->
+                    not_found when Create ->
```

Comment:
So the intent here is to crash on reopen rather than return an undefined state?

Reply:
yeah, I didn't see a good way to return the error, but I can work it out so that things don't crash here.  This was just to flush the error to the surface.